powerflow/fault_check.cpp

```diff
@@ -32,6 +32,7 @@ fault_check::fault_check(MODULE *mod) : powerflow_object(mod)
 				PT_KEYWORD, "ONCHANGE", (enumeration)ONCHANGE,
 				PT_KEYWORD, "ALL", (enumeration)ALLT,
 				PT_KEYWORD, "SINGLE_DEBUG", (enumeration)SINGLE_DEBUG,
+				PT_KEYWORD, "SWITCHING", (enumeration)SWITCHING,
```

Comment:
The "SWITCHING" flag seems to be mostly to get any topology changes to be captured when an eventgen object isn't present.  fault_check should work without eventgen objects regardless, so maybe this was built on an earlier assumption?